### PR TITLE
Some tree fixes

### PR DESCRIPTION
**Pull request by danielhers**
I'm new to UD annotation, but I think I found some errors. Are my corrections any good?

### CODE REVIEW

File: not-to-release/sources/reviews/241739.xml.conllu [hunk 1]
```diff
@@ -59,7 +59,7 @@
 15	pay	pay	VERB	VB	VerbForm=Inf	4	advcl	4:advcl:because	_
 16	attention	attention	NOUN	NN	Number=Sing	15	obj	15:obj	_
 17	to	to	ADP	IN	_	18	case	18:case	_
-18	detail	detail	NOUN	NN	Number=Sing	16	nmod	16:nmod:to	SpaceAfter=No
+18	detail	detail	NOUN	NN	Number=Sing	15	nmod	15:obl:to	SpaceAfter=No
```
Comment (nschneid) [inline not-to-release/sources/reviews/241739.xml.conllu]:
```suggestion
18	detail	detail	NOUN	NN	Number=Sing	15	obl	15:obl:to	SpaceAfter=No
```

**Comment by nschneid** [comment]
Thanks @danielhers, but fixes should go in the source documents under not-to-release/sources. The main train/dev/test files will be generated from those.

**Comment by danielhers** [comment]
Oh, right. I'll move my changes there.

File: not-to-release/sources/reviews/241739.xml.conllu [hunk 2]
```diff
@@ -59,7 +59,7 @@
 15	pay	pay	VERB	VB	VerbForm=Inf	4	advcl	4:advcl:because	_
 16	attention	attention	NOUN	NN	Number=Sing	15	obj	15:obj	_
 17	to	to	ADP	IN	_	18	case	18:case	_
-18	detail	detail	NOUN	NN	Number=Sing	16	nmod	16:nmod:to	SpaceAfter=No
+18	detail	detail	NOUN	NN	Number=Sing	15	<span class="x x-first x-last">obl</span>	15:obl:to	SpaceAfter=No
```
Comment (nschneid) [inline not-to-release/sources/reviews/241739.xml.conllu]: It looks like the beta-quality GitHub feature I used to suggest a change is buggy and inserted HTML. :P

Comment (danielhers) [reply not-to-release/sources/reviews/241739.xml.conllu]: Yeah, weird. I'm replacing these commits anyway though.

File: not-to-release/sources/reviews/292841.xml.conllu [hunk 3]
```diff
@@ -145,11 +145,12 @@
...
 14	,	,	PUNCT	,	_	17	punct	17:punct	_
 15	and	and	CCONJ	CC	_	17	cc	17:cc	_
+15.1	's	be	VERB	VBZ	_	_	_	2:conj:and	CopyOf=2
 16	a	a	DET	DT	Definite=Ind|PronType=Art	17	det	17:det	_
-17	market	market	NOUN	NN	Number=Sing	4	conj	2:nsubj|4:conj:and	_
+17	market	market	NOUN	NN	Number=Sing	4	conj	2:nsubj|4:conj:and|15.1:nsubj	_
 18	across	across	ADP	IN	_	20	case	20:case	_
 19	the	the	DET	DT	Definite=Def|PronType=Art	20	det	20:det	_
-20	street	street	NOUN	NN	Number=Sing	17	nmod	17:nmod:across	SpaceAfter=No
+20	street	street	NOUN	NN	Number=Sing	17	orphan	15.1:advmod:across	SpaceAfter=No
```
Comment (nschneid) [inline not-to-release/sources/reviews/292841.xml.conllu]: Adverbial PPs are always obl, I believe, not advmod.

**Pull request merged**